### PR TITLE
Support for PHP scope

### PR DESCRIPTION
**Pull request by vool**
Hi,
I am using Talisman on a PHP project, the composer lock file contains lots of hashes which are always ok as it is autogenerated.
Would be great to add support for PHP scope.
Keith

### CODE REVIEW

File: talismanrc/scopes.go [hunk 1]
```diff
@@ -6,4 +6,5 @@ var knownScopes = map[string][]string{
 	"images":    {"*.jpeg", "*.jpg", "*.png", "*.tiff", "*.bmp"},
 	"bazel":     {"*.bzl"},
 	"terraform": {".terraform.lock.hcl"},
+	"php":       {"composer.lock", "vendor/"},
```
Comment (svishwanath-tw) [inline talismanrc/scopes.go:9]: Hi Keith,
I worry that vendor folder if checked in could become a source of secret leakage.
How is 'no modifications to vendor' enforced' ?
Why not git ignore it and install on developer machines through some script or out of band sharing ?

**Review by svishwanath-tw — CHANGES_REQUESTED** [review]

**Comment by svishwanath-tw** [comment]
I've modified the existing golang scope to remove the vendor folder.
I'm unsure if node_modules belongs in the same league.

File: talismanrc/scopes.go (hunk 1 above)
Comment (vool) [reply talismanrc/scopes.go]: Hi Suhas,
You are right, the vendor directory is normally .gitignore'd, was following the same pattern as go and node.
Have updated the PR to exclude the vendor dir.
Keith

**Review by svishwanath-tw — APPROVED** [review]

**Comment by svishwanath-tw** [comment]
@vool : Please resolve merge conflict. (preferably rebase against `thoughtworks:master` or at least merge `thoughtworks/master` into `vool:master`)

**Comment by vool** [comment]
Already up to date.


> @vool : Please resolve merge conflict. (preferably rebase against `thoughtworks:master` or at least merge `thoughtworks:master` into `vool:master`)

**Pull request merged**